evergreen/evergreen.yml

```diff
@@ -417,15 +417,16 @@ functions:
           ${PREPARE_SHELL}
           OCSP_TLS_SHOULD_SUCCEED="${OCSP_TLS_SHOULD_SUCCEED}" \
             OCSP_ALGORITHM=${OCSP_ALGORITHM} \
-            evergreen/add-certs.sh
+            SSL="ssl" \
+            evergreen/add-certs-if-needed.sh
           set +o xtrace
           AUTH="${AUTH}" \
             SSL="ssl" \
             TOPOLOGY="${TOPOLOGY}" \
             MONGODB_URI="${MONGODB_URI}" \
             OCSP_TLS_SHOULD_SUCCEED="${OCSP_TLS_SHOULD_SUCCEED}" \
               evergreen/run-tests.sh
-          evergreen/remove-certs.sh
```

Comment:
should this script be removed then?

Reply:
Hm, we could. I still find this script useful when testing locally, and leaving it in makes restoring this behavior fairly easy, so I'd prefer to leave it in for now I think :)

Reply:
I think having scripts that we don't use at all can be confused. At some moment, we can forget that this script is not used by design and we will need to research where this script is used and it can take a time since the answer is it's used nowhere :) 
We will be able to find this script in git history at any moment.

I won't insist, I will rely on your and other reviewers opinion here

Reply:
I don't mind removing it, removed!